mathesar/database/types/types.py

```diff
@@ -0,0 +1,5 @@
+from mathesar.database.types import email
```

Comment:
I think this would make sense in `__init__.py`

Reply:
Agreed

Reply:
I still think the contents of this file would make  sense in `__init__.py` instead of importing it into there.